src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public String getRandomBall() {
+        Ball ball = new Ball();
+        ColorSupplier color = new ColorSupplier();
+        ball.setNumber(new Random().nextInt(100));
```

Comment:
All magic numbers in your code should be constants. Please define a constant for the number 100, which represents the upper limit for the random number generation.